File: Vishwas RNO 40/DotNet/Assignment 2/DemoApp/hospital/Patient.cs
namespace hospital;

class Patient
{
    public int PatientId { get; set;}
    public string PatientName { get; set;}
    public int BedType { get; set;}
    public int NoOfdays { get; set;}

    public Patient()
    {
        this.PatientId = 1;
        this.PatientName = "Sujit";
        this.BedType = 1;
        this.NoOfdays = 5;
	}
   public Patient(int PatientId, string PatientName, int BedType, int NoOfdays)
   {
        this.PatientId = PatientId;
        this.PatientName = PatientName;
        this.BedType = BedType;
        this.NoOfdays = NoOfdays;
   } 

   	public virtual double GetBillAmount(){
		double price= 0;
		double amount;
	if(BedType == 1)
		price = 500;
	if(BedType == 2)
		price =350;
	if(BedType == 3)
		price =200;
	amount = NoOfdays *price;
	return amount;
	}
    


}

File: Kapil RNO 39/Java/Assign3/src/Patient.java
package hospital;

public class Patient {
	private int patientId;
	private String patientName;
	private int bedType;
	private int noOfDays;

	public Patient(int pId, String pName, int bed, int days){
		patientId= pId;
		patientName= pName;
		bedType= bed;
		noOfDays= days;
	}
	
	public Patient() {
		this(101, "Sujit", 1, 5);
	}
	
	public int getPatientId() {
		return patientId;
	}
	public void setPatientId(int ii){
		patientId= ii;
	}

	public String getPatientName() {
		return patientName;
	}
	public void setPatientName(String name){
		patientName= name;
	}

	public int getBedType() {
		return bedType;
	}
	public void setBedType(int bed){
		bedType= bed;
	}

	public int getNoOfDays() {
		return noOfDays;
	}
	public void setNoOfDays(int days){
		noOfDays= days;
	}

	public void printEmployee(){
		System.out.printf(" Employee ID : %d%n",patientId);
		System.out.printf(" Employee Name : %s%n",patientName);
		System.out.printf(" Employee Bed Type : %d%n",bedType);
		System.out.printf(" Number of days : %d%n",noOfDays);
	}
	public double getBillAmount(){
		double price= 0;
		double amount;
	if(bedType == 1)
		price = 500;
	if(bedType == 2)
		price =350;
	if(bedType == 3)
		price =200;
	amount = noOfDays *price;
	return amount;
	}

}
